### PR TITLE
ZCS-9073: Support 2FA login for Admin Console Frontend

### PR DESCRIPTION
**Pull request by aayushvijayvergiya**

### CODE REVIEW

File: WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js [hunk 1]
```diff
@@ -183,7 +183,7 @@ function(child, childHtmlElement) {
 
 ZaLoginDialog.prototype._loginSelListener =
 function() {
-	this.setCursor("wait");
+	// this.setCursor("wait");
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js:186]: fix this

Comment (aayushvijayvergiya) [reply WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js]: Done

File: WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js [hunk 2]
```diff
@@ -183,7 +183,6 @@ function(child, childHtmlElement) {
 
 ZaLoginDialog.prototype._loginSelListener =
 function() {
-	this.setCursor("wait");
```
Comment (BhavinBathani) [inline WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js:186]: Don't we need this?

**Review by BhavinBathani — APPROVED** [review]
LGTM

File: WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js (hunk 2 above)
Comment (aayushvijayvergiya) [reply WebRoot/js/zimbraAdmin/common/ZaLoginDialog.js:186]: We are already showing `Loading...` text. Also whenever the request was failing, it was going in an infinite state. I think its okay to remove it.

**Review by madhavdhuppe — APPROVED** [review]

**Pull request merged**